content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -54,19 +58,30 @@ $excerpt-line-height: 20;
 
     .context,
     .source {
+      @include dark-theme-only {
+        color: $teal-10;
```

Comment:
@Mardak This styling isn't in _theme.scss so it does feel like a specific request from design to have a colour not already defined. Is the idea here we add something to theme.scss for this text to swap from teal-80/teal-10?